harness/determined/ipc.py

```diff
@@ -713,7 +719,9 @@ def health_check() -> None:
                 except subprocess.TimeoutExpired:
                     logging.error(f"killing worker which didn't exit after {on_exit.name}")
                     p.send_signal(signal.SIGKILL)
-        return p.wait()
+        exit_code = p.wait()
+        logging.debug(f"exit_code: {exit_code}")
```

Comment:
if you leave this in, you'll have to provide more information on where this log line is coming from.